docs/quick-answers/linode-platform/cloud-manager-search/index.md

```diff
@@ -13,80 +13,98 @@ contributor:
+
+For example, to search for all images on your account, you would enter the following:
 
     type:image
 
-And to find all applicable resources created in the year 2020:
+To find all applicable resources created in the year 2020:
 
     created:2020
 
-## Standard Boolean Operators
+You can make this more focused by combining the standard search with the keywords. For example, the following query searches for NodeBalancers with the label or tag ecommerce:
 
-The Cloud Manager includes support for standard boolean operators `OR` and `AND` which can help to filter the results of your search further. `OR` can be used to search for multiple different keywords in a single query  and `AND` will allow you to narrow your search more precisely for additional qualifiers. For example, in the following query, we'd like to search for ALL of the Linodes and Images on our account:
+    type:NodeBalancer ecommerce
 
-    type:linode OR type:image
+## Standard Logical Operators
 
-And in the following we'll try to find all of our Linodes which ALSO match our `production` tag:
+The Cloud Manager includes support for standard logical operators `OR`, `AND`, and `-` (NOT) which can help to filter the results of your search further.
 
-    type:linode AND tag:production
+### OR
 
-## Negate Operators
+`OR` can be used to search for multiple different keywords in a single query. For example, use the following query to display all Linodes and Images on your account:
 
-A negate operator can be used to return all values that are **NOT** defined by the negate operator. This will always be preceded by the `-` symbol. For example, see the following query:
+    type:linode OR type:image
 
-    -type:linode
+### AND
 
-When used, you will see all Linodes excluded from your query.
+`AND` will allow you to narrow your search more precisely for additional qualifiers. The following search finds all of th Linodes which also match the `production` tag:
 
-## Nested Queries
+    type:linode AND tag:production
 
-Nested Queries can also be performed for an even higher level of control. If you want to search for all of your domains, as well as all of your Linodes that match the tag `my-tag`for example, you can use the following Nested Query:
+### NOT
 
-    type:domain OR (type:linode AND tag:my-tag)
+The **NOT**, or negate, operator is used by preceding the keyword with a `-` symbol. A negate operator can be used to return all values that are **NOT** defined by the negate operator. For example, this query will return everything except Linodes:
+
+    -type:linode
 
 ## Aliases
 
-In addition most operators include a number of aliases that you can use as an alternative to standard usage. All aliases and their standard equivalent can be seen in the chart below.
+Some operators and keywords include one or more aliases that you can use as an alternative to standard usage. Aliases and their standard equivalent(s) are in the chart below.
```

Comment:
some of these are operators and some keywords, likewise, some just have one, some more.